frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -165,8 +180,39 @@ export function ProjectList() {
                     <ChevronsUpDown className="w-4" />
                   </MenubarTrigger>
                   <MenubarContent className="border w-[calc(var(--radix-menubar-trigger-width)+0.5rem)] border-border md:min-w-6 md:w-auto">
-                    <MenubarContentItem icon={Rocket} text="Active" />
-                    <MenubarContentItem icon={Trash} text="Archived" />
+                    <div
+                      onClick={() =>
+                        navigate({
+                          search: {
+                            slug,
+                            page: 1,
+                            per_page,
+                            sort_by,
+                            status: "active"
+                          },
+                          replace: true
+                        })
+                      }
+                    >
+                      <MenubarContentItem icon={Rocket} text="Active" />
+                    </div>
```

Comment:
Shouldn't these be like this instead ? 

```tsx
<MenubarContentItem onClick={() => navigate({...})} icon={...} text={...} />
```

Also you can simplify navigate with : 

```tsx
 () => navigate({...filters, page: 1, status: 'active' })
```